tests/testutil.h

```diff
@@ -201,12 +201,12 @@ static void NN_UNUSED test_drop_impl (char *file, int line, int sock, int err)
     }
 }
 
-static int get_test_port(int argc, const char *argv[])
+static int NN_UNUSED get_test_port (int argc, const char *argv[])
```

Comment:
What is unused here?  The return value?  The arguments?  Which ones?  I think you need to apply NN_UNUSED in front of specific arguments if that's what they are for.